### ISSUE TITLE
[4.0.0-beta0] TypeError: Cannot read property 'column' of undefined

### ISSUE THREAD

**Issue by roderik** [trufflesuite/truffle#609]
## Issue

I have a failing transaction

```
Running migration: 3_initial_coins.js
  Running step...
  ... 0xcc0b0a8af92515bb72861fa413210e954fb6b4b7e77293734afa1992d68bef4a
Error encountered, bailing. Network state unknown. Review successful transactions manually.
Error: VM Exception while processing transaction: invalid opcode
```

So I try to use the debug:

```
truffle(develop)> debug 0xcc0b0a8af92515bb72861fa413210e954fb6b4b7e77293734afa1992d68bef4a
Gathering transaction data...

Addresses affected:
  0x818d164806517b4edbbf290a4ddd78a26503481d - UnrestrictedCoinFactory

Commands:
(enter) last command entered (step next)
(o) step over, (i) step into, (u) step out, (n) step next
(;) step instruction, (p) print instruction, (h) print this help, (q) quit


UnrestrictedCoinFactory.sol | 0x818d164806517b4edbbf290a4ddd78a26503481d:

5: import './UnrestrictedCoinRegistry.sol';
6: 
7: contract UnrestrictedCoinFactory is Ownable {
TypeError: Cannot read property 'column' of undefined
```
and the debug session ends.

**Comment by Systerr** [comment]
I can reproduce this bug on 4.0.0-beta1

**Comment by androolloyd** [comment]
i uninstalled and did a fresh install after receiving this error and was presented with another error, around a diff variable not being defined

**Comment by gnidan** [comment]
@roderik @Systerr @androolloyd does the problem still persist for you if you delete your `build/contracts/` directory and recompile?

If any of you have code exhibiting this issue on a public repo, I'd love to be able to reproduce it locally!

**Comment by gnidan** [comment]
Followup question: What version of the TestRPC are you all using?

**Comment by roderik** [comment]
Well, the built in one. truffle develop will start one on it's own

**Comment by androolloyd** [comment]
Gathering transaction data...

/usr/local/lib/node_modules/truffle/build/cli.bundled.js:309080
      accept(result.result.structLogs);
                          ^

TypeError: Cannot read property 'structLogs' of undefined
    at /usr/local/lib/node_modules/truffle/build/cli.bundled.js:309080:27
    at /usr/local/lib/node_modules/truffle/build/cli.bundled.js:294942:9
    at XMLHttpRequest.request.onreadystatechange (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:296367:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:164934:18)
    at XMLHttpRequest._setReadyState (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:165224:12)
    at XMLHttpRequest._onHttpResponseEnd (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:165379:12)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:165339:24)
    at emitNone (events.js:110:20)
    at IncomingMessage.emit (events.js:207:7)
    at endReadableNT (_stream_readable.js:1059:12)

**Comment by androolloyd** [comment]
this is with a geth dev node

**Comment by Systerr** [comment]
Hi. @gnidan i able to reproduce the same problem after deleting build directory and recompile.
truffle@4.0.0-beta.2
ethereumjs-testrpc@4.1.3
Sorry right now source code is closed

**Comment by gnidan** [comment]
This has been fixed via a workaround and released in v4.0.1. The debugger should no longer fail/quit on error:

> TypeError: Cannot read property 'column' of undefined

The underlying bug seems to still exist, leaving ticket open to investigate.
